stage/tests/lms/test_dashboard.py

```diff
@@ -0,0 +1,26 @@
+from bok_choy.web_app_test import WebAppTest
+from ...pages.lms.login_lms import LmsLogin
+from ...pages.lms.dashboard_lms import DashboardPageExtended
+from ...pages.lms.course_page_lms import CourseInfoPageExtended
+from edxapp_pages.lms.courseware import CoursewarePage
+import os
```

Comment:
nit: `import` statements should be on the top of the document, with `from x import y` below that.